### PR TITLE
data, first functions, test added

### PR DESCRIPTION
**Pull request by oscar-defelice**
I have opened a PR for you @garghi2000.

in this way we can start with the code review.

### CODE REVIEW

File: test_Current_jumps.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from Current_jumps import std_1D
+from Current_jumps import get_n_moms_of_moving_array
+import pytest
+
+# list of inputs and expected results for std_1D() function
+def list_to_test_std_1D():
```
Comment (oscar-defelice) [inline test_Current_jumps.py:7]: This function is not a test, hence I would put that in a module called maybe test_utils and then import in this file

File: test_Current_jumps.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
```
Comment (oscar-defelice) [inline test_Current_jumps.py:1]: In general the test modules thay have to go into the `tests/` directory.

File: Current_jumps.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
+import numpy as np
+import pandas as pd
+import math
+      
+def standardization(array):
```
Comment (oscar-defelice) [inline Current_jumps.py:5]: missing docstrings in all the functions 😄

File: src/current_jumps.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import pandas as pd
+import math
+      
+def standardization(array):
```
Comment (oscar-defelice) [inline src/current_jumps.py:5]: As before, missing docstring

File: tests/test_current_jumps.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from src.current_jumps import std_1D
+from src.current_jumps import get_n_moms_of_moving_array
+import pytest
+
+# list of inputs and expected results for std_1D() function
+def list_to_test_std_1D():
```
Comment (oscar-defelice) [inline tests/test_current_jumps.py:7]: these functions are not test, hence they should go into a subdirectory called test_utils, and imported from there.

**Review by garghi2000 — APPROVED** [review]

**Pull request merged**